Library/Edge.cs

```diff
@@ -92,9 +105,10 @@ public void SetPrev(Vertex v, Edge other)
             else prev2 = other;
         }
 
-        /**
-         * Return all faces that use this edge as a side.
-         */
+        /// <summary>
+        /// Returns all <see cref="Face"/>s that use the <see cref="Edge"/> as a side.
+        /// </summary>
+        /// <returns>all <see cref="Face"/>s that use the <see cref="Edge"/> as one of it's sides.</returns>
```

Comment:
idk if it's worth repeating the same thing in summary and return, what is the standard practice in C# in your opinion?

Reply:
> idk if it's worth repeating the same thing in summary and return, what is the standard practice in C# in your opinion?

I kind of feel the same way. But looking at a bunch of the docs of the built in C# code it seems to be the case. I'm not opposed to not doing it though.